semanticdb-java/src/main/protobuf/semanticdb.proto

```diff
@@ -178,4 +190,93 @@ message ExistentialType {
   reserved 2;
   Type tpe = 1;
   Scope declarations = 3;
+}
+
+message Tree {
+  oneof sealed_value {
+    ApplyTree apply_tree = 1;
+    IdTree id_tree = 3;
+    LiteralTree literal_tree = 4;
+    OriginalTree original_tree = 6;
+    SelectTree select_tree = 7;
+    // -- OUT OF SPEC -- //
+    AnnotationTree annotation_tree = 8;
+    // -- OUT OF SPEC -- //
+  }
+}
+
+message ApplyTree {
+  Tree function = 1;
+  repeated Tree arguments = 2;
+}
+
+message IdTree {
+  string symbol = 1;
+}
+
+message OriginalTree {
+  Range range = 1;
+}
+
+message LiteralTree {
+  Constant constant = 1;
+}
+
+message SelectTree {
+  Tree qualifier = 1;
+  IdTree id = 2;
+}
+
+message AnnotationTree {
+  Annotation annotation = 1;
```

Comment:
> It's worth discussing if we should replace SymbolInformation.annotation with AnnotationTree down the road.

This seems like a good idea to me, seeing as `AnnotationTree` has a single `Annotation` field. If we flatten `AnnotationTree`, then it would be a benefit to both [`AnnotatedType`](https://scalameta.org/docs/semanticdb/specification.html#type) and [`SymbolInformation`](https://scalameta.org/docs/semanticdb/specification.html#symbolinformation). This _should_ be backwards compatible (unless changing message name is not), so I dont see any reason why not to do this :slightly_smiling_face: Would also mean no duplication between `Annotation` and `AnnotationTree`